alg-excer1-palindrome.js

```diff
@@ -0,0 +1,15 @@
+function palindrome(input) {
+  str =  input.toString().toLowerCase();
+  var strLength= str.length;
+
+  for (var i=0; i<strLength/2; i++) {
+    if (str[i] !== str[strLength-1-i]) {
+      return input + "- is not Palindrome";
+    }
+  }
+  return input + "- is Palindrome";
+}
+
+console.log(palindrome("SA as"));
+
+console.log(palindrome("345543"));
```

Comment:
create a function to test your results.

you need to think of more edge cases.

Reply:
updated